superset-frontend/src/explore/components/controls/VizTypeControl/index.tsx

```diff
@@ -365,41 +461,56 @@ const VizTypeControl = (props: VizTypeControlProps) => {
           <VizSupportValidation vizType={initialValue} />
         </>
       </Tooltip>
+
       <UnpaddedModal
         show={showModal}
         onHide={toggleModal}
         title={t('Select a visualization type')}
         primaryButtonName={t('Select')}
         onHandledPrimaryAction={onSubmit}
+        maxWidth="1090px"
         responsive
       >
         <VizPickerLayout>
-          <SearchPane>
-            <Input
-              ref={searchRef}
-              type="text"
-              value={filter}
-              placeholder={t('Search')}
-              onChange={changeSearch}
-              data-test={`${VIZ_TYPE_CONTROL_TEST_ID}__search-input`}
-            />
-          </SearchPane>
-          <CategoriesTabs tabPosition="left">
-            {Object.entries(categories).map(([category, vizTypes]) => (
-              <Tabs.TabPane tab={category} key={category}>
-                <IconPane
-                  data-test={`${VIZ_TYPE_CONTROL_TEST_ID}__viz-row`}
-                  gutter={16}
-                >
-                  {vizTypes.map(entry => (
-                    <Col xs={12} sm={8} md={6} lg={4} key={entry.key}>
-                      {renderItem(entry)}
-                    </Col>
-                  ))}
-                </IconPane>
-              </Tabs.TabPane>
+          <LeftPane>
+            <SearchWrapper>
+              <Input
+                type="text"
+                value={searchInputValue}
+                placeholder={t('Search')}
+                onChange={changeSearch}
+                onFocus={startSearching}
```

Comment:
I think we should display all values when no category is selected or no filter is applied, independently of the search input focus.

Examples:
- User clicks on search input. Show all values.
- User types empty. Show all values.
- User types valid text. Show filtered values.
- User focuses out of input without selecting a category or entering a text. Show all values.
- User focuses out of input without selecting a category but entering a text. Show filtered values.